getting-started/set-up-services.hbs.md

```diff
@@ -37,22 +37,25 @@ Before following this walkthrough, you must:
 
 1. Have access to a cluster with Tanzu Application Platform installed.
 1. Have downloaded and installed the Tanzu CLI and the corresponding plug-ins.
-1. Ensure your Tanzu Application Platform cluster can pull the images required by the RabbitMQ Cluster Kubernetes operator. For more information, see the [RabbitMQ documentation](https://www.rabbitmq.com/kubernetes/operator/using-operator.html).
+1. Ensure your Tanzu Application Platform cluster can pull the images required by the service operator. For more information, see the following documentation:
```

Comment:
*nitpick*: Might be good to be more specific here w/r to the term _"images"_. I know this is nothing that has ben introduced by this PR and was there before, still it caught my eye when reading over this.

```suggestion
1. Ensure your Tanzu Application Platform cluster can pull the container images required by the service operator. For more information, see the following documentation:
```